### ISSUE TITLE
WCAG 1.4.3: Elements must have sufficient color contrast (.learnmore[aria-label="Learn\ more"][target="_blank"])

### ISSUE THREAD

**Issue by jwunderl** [microsoft/pxt-microbit#2320]
#### Issue
`Elements must have sufficient color contrast` ([color-contrast](https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=msftAI))
#### Target application
[Microsoft MakeCode for micro:bit](http://localhost:3232/index.html#editor)
#### Element path
.card.link[role="link"]:nth-child(32) > .extra.hide.mobile > .learnmore[aria-label="Learn\ more"][target="_blank"]
#### Snippet
`<a class="learnmore right floated" href="/pkg/tinkertanker/pxt-ringbitcar" aria-label="Learn more" target="_blank" rel="noopener noreferrer">Learn more</a>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 2.84 (foreground color: #999999, background color: #ffffff, font size: 12.0pt, font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 75.0.3770.142
This accessibility issue was found using Accessibility Insights for Web 2.7.1 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).

<img width="169" alt="Screen Shot 2019-07-26 at 3 49 04 PM" src="https://user-images.githubusercontent.com/5615930/61985194-f6276480-afbc-11e9-9ff2-a4befb2ec0ce.png">
